operator/docs/lokistack/object_storage.md

```diff
@@ -152,35 +152,82 @@ Loki Operator supports [AWS S3](https://aws.amazon.com/), [Azure](https://azure.
 
 ### Requirements
 
-* Deploy the [OpenShift Data Foundation](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10) on your cluster.
-
-* Create a bucket via an ObjectBucketClaim.
+* Deploy the [OpenShift Data Foundation](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12) on your cluster.
 
 
 ### Installation
 
 * Deploy the Loki Operator to your cluster.
+* Create an ObjectBucketClaim in `openshift-logging` namespace:
 
-* Create an Object Storage secret with keys as follows:
+  ```yaml
+    apiVersion: objectbucket.io/v1alpha1
+    kind: ObjectBucketClaim
+    metadata:
+      name: loki-bucket-odf
+      namespace: openshift-logging
+    spec:
+      generateBucketName: loki-bucket-odf
+  ```
 
-    ```console
-    kubectl create secret generic lokistack-dev-odf \
-      --from-literal=bucketnames="<BUCKET_NAME>" \
-      --from-literal=endpoint="https://s3.openshift-storage.svc" \
-      --from-literal=access_key_id="<ACCESS_KEY_ID>" \
-      --from-literal=access_key_secret="<ACCESS_KEY_SECRET>"
-    ```
+* Get bucket properties from the associated ConfigMap:
+  ```console
+  BUCKET_HOST=$(oc get -n openshift-logging configmap loki-bucket-odf -o jsonpath='{.data.BUCKET_HOST}')
+  BUCKET_NAME=$(oc get -n openshift-logging configmap loki-bucket-odf -o jsonpath='{.data.BUCKET_NAME}')
+  BUCKET_PORT=$(oc get -n openshift-logging configmap loki-bucket-odf -o jsonpath='{.data.BUCKET_PORT}')
+  ```
 
-    where `lokistack-dev-odf` is the secret name. You can copy the values for `BUCKET_NAME`, `ACCESS_KEY_ID` and `ACCESS_KEY_SECRET` from your ObjectBucketClaim's accompanied secret.
+* Get bucket access key from the associated Secret:
+  ```console
+  ACCESS_KEY_ID=$(oc get -n openshift-logging secret loki-bucket-odf -o jsonpath='{.data.AWS_ACCESS_KEY_ID}' | base64 -d)
+  SECRET_ACCESS_KEY=$(oc get -n openshift-logging secret loki-bucket-odf -o jsonpath='{.data.AWS_SECRET_ACCESS_KEY}' | base64 -d)
+  ```
+
+  * Create an Object Storage secret with keys as follows:
+
+  ```console
+    oc create -n openshift-logging secret generic lokistack-dev-odf \
+    --from-literal=access_key_id="${ACCESS_KEY_ID}" \
+    --from-literal=access_key_secret="${SECRET_ACCESS_KEY}" \
+    --from-literal=bucketnames="${BUCKET_NAME}" \
+    --from-literal=endpoint="https://${BUCKET_HOST}:${BUCKET_PORT}"
+  ```
+
+  Where `lokistack-dev-odf` is the secret name. The values for `ACCESS_KEY_ID`, `SECRET_ACCESS_KEY`, `BUCKET_NAME`, `BUCKET_HOST` and `BUCKET_PORT` are taken from your ObjectBucketClaim's accompanied secret and ConfigMap.
 
 * Create an instance of [LokiStack](../hack/lokistack_dev.yaml) by referencing the secret name and type as `s3`:
```

Comment:
The link here is also broken, it should be `../../hack/lokistack_gateway_ocp.yaml`. Would you mind adding this fix into this PR please?